scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -530,6 +532,8 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             File.Delete(compilersDeps);
             File.Delete(compilersRuntimeConfig);
 
+            CrossgenUtil.CrossgenDirectory(c, outputDir);
```

Comment:
Does crossgenn'ing Stage1 take a lot of time?  If we only crossgenn'd Stage2, would that save some time in our build?